libs/api-bot/src/Network/Wire/Bot/Monad.hs

```diff
@@ -858,6 +858,7 @@ randUser (Email loc dom) (BotTag tag) = do
         , newUserPict           = Nothing
         , newUserAssets         = []
         , newUserAccentId       = Nothing
+        , newUserEmailCode      = Nothing
```

Comment:
where's this coming from?

Reply:
This new parameter was introduced [back here](https://github.com/wireapp/wire-server/commit/03aa0c7d05ce791c8909f78a62e2b14626351e6e)